### PR TITLE
feat: box-shadow background

### LINKED ISSUE #833: Intro.js doesn't position right when using transform: translate3d
## Description
I found it impossible to position intro.js correctly on leaflet map dots and isolated the issue to being because they use translate3d.

## Expected Behavior
Correct placement of annotations

## Actual Behavior
Wrong placement on DOM elements with transform: translate3d(x,y,0)

## Example (recommended)
https://codepen.io/anon/pen/VXjoOQ

This is an overly simplified example, but if you comment out the line above transform: translate3d and try with top and left defined, you will see that it's working when not using transform: translate3d.

## Environment (optional)

Tested with latest chrome, firefox, latest intro.js, windows 10

### PR DESCRIPTION
**Pull request by afshinm**
This PR removes the background-color from `introjs-overlay` and instead, uses `box-shadow` to provide the same feature. 

Let me know what you think @bozdoz 

Fixes #833, #135, #109

### CODE REVIEW

**Comment by afshinm** [comment]
@bozdoz here is the live example https://intro-js-3lymw5rly.vercel.app/example/hello-world/index.html

**Comment by afshinm** [comment]
@bozdoz also the Netlify app https://deploy-preview-1075--introjs.netlify.app/example/

File: src/util/createElement.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /**
  * Create a DOM element with various attributes
- *
+ *
```
Comment (bozdoz) [inline src/util/createElement.js:3]: Could help to add husky to the project and run prettier automatically on every commit

Comment (afshinm) [reply src/util/createElement.js:3]: I like that idea @bozdoz

File: src/styles/introjs.scss [hunk 2]
```diff
@@ -97,10 +82,9 @@ tr.introjs-showElement {
   box-sizing: content-box;
   position: absolute;
   z-index: 9999998;
-  background-color: rgba($white, 0.9);
   border: 1px solid rgba($black900, 0.5);
   border-radius: 4px;
-  box-shadow: 0 2px 15px rgba($black900, 0.4);
+  box-shadow: 0 0 1px 2px rgba($black900, 0.8), rgba($black900, 0.4) 0 0 0 5000px;
```
[the remark this answers is not in the document]
Comment (afshinm) [reply src/styles/introjs.scss]: The first box-shadow is the helperLayer border, the second box shadow is the overlay background the covers the entire screen.

**Comment by afshinm** [comment]
@bozdoz just added Jest and some basic tests.

File: src/styles/introjs.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (bozdoz) [reply src/styles/introjs.scss]: Right, but why add a second "border"?

[the remark this answers is not in the document]
Comment (afshinm) [reply src/styles/introjs.scss]: oh sorry, I misread your comment. I will remove the border.

[the remark this answers is not in the document]
Comment (afshinm) [reply src/styles/introjs.scss]: @bozdoz removed

**Comment by bozdoz** [comment]
LGTM

**Review by bozdoz — APPROVED** [review]

**Comment by bozdoz** [comment]
Tested for stacking context problem (parent has position: absolute, z-index, perspective, and transform: rotate(1deg)):

Tested current live site:
image

Tested Netlify deployment:
image

**Comment by afshinm** [comment]
\o/

thanks for the review @bozdoz!

**Pull request merged**